### ISSUE TITLE
关于BNCI2015数据集复现的疑问

### ISSUE THREAD

**Issue by ejejeieei** [staraink/MIRepNet#7]
作者您好，十分感谢您的工作。我现在在BNCI2015001数据集上进行复现的性能与论文的报告有较大差距，特别是被试9，10的差距会异常大。请问这可能是哪些原因导致的？能否提供各个数据集session选择和更加详细的超参数配置？谢谢

<img width="1722" height="115" alt="Image" src="https://github.com/user-attachments/assets/7cf12037-eef1-468e-b6b5-6c2c99db4b60" />

**Comment by Dingkun0817** [comment]
Hi, thank you very much for your interest in our work and for trying to reproduce the results.

For BNCI2015001, we use only **the first session** in our experiments. As for the training setup, the reported results can be reproduced with the default hyperparameter configuration in the released code.

The unusually large gap on subjects 9 and 10 may come from details such as session selection, preprocessing consistency, random seed, or other implementation settings. So I would first suggest checking whether the same session split and preprocessing pipeline are being used.

Thank you again for your careful reproduction.
